src/librustdoc/html/render/print_item.rs

```diff
@@ -1322,6 +1341,15 @@ where
     w.write_str("</div>")
 }
 
+fn wrap_item<F>(w: &mut Buffer, item_name: &str, f: F)
```

Comment:
Great idea! :+1:

Reply:
I simply borrowed it from `wrap_into_docblock` above :)